ql/src/java/org/apache/hadoop/hive/ql/security/authorization/command/CommandAuthorizerV2.java

```diff
@@ -158,6 +158,15 @@ private static boolean isDeferredAuthView(Table t){
     if (TableType.MATERIALIZED_VIEW.name().equals(tableType) || TableType.VIRTUAL_VIEW.name().equals(tableType)) {
       isView = true;
     }
+    if (isView) {
+      Map<String, String> params = t.getParameters();
+      if (params != null && params.containsKey(authorizedKeyword)) {
```

Comment:
nice; so HIVE-25695 effectively mangled the [original condition to be spark dependent](https://github.com/apache/hive/commit/e61460b36178de3d8f2ddb28ad3f03902a34c3a8#diff-b1bce663288d98098d8aaa474ad0694d43484b0cd1977a788dc9f6c6f7fae4d0R163)...

Reply:
In theory this could be used by any other system.
In reality, yes :(